### PR TITLE
base.phone

### PR DESCRIPTION
**Pull request by carresemata**
Everything within margins. Have 50% less phones because we also reduced the num of offices

### CODE REVIEW

File: migration_original/ODS1Stage/tables/Base/Phone/spu_translated_Phone.sql [hunk 1]
```diff
@@ -289,100 +359,140 @@ cte_tmp_phones as (
...
-        left join CTE_Tmp_Phones as cte on cte.phonenumber = p.phonenumber
-    where cte.displaypartnercode = 'HG'
-)
+    )
 select 
-    uuid_string() as PhoneId,
-    PhoneNumber
-from CTE_TMP_PHONES
-where not exists (select * from CTE_NOTEXISTS) $$;
+    distinct
```
Comment (ASanchezXtillion) [inline migration_original/ODS1Stage/tables/Base/Phone/spu_translated_Phone.sql:371]: no distinct

File: migration_original/ODS1Stage/tables/Base/Phone/spu_translated_Phone.sql [hunk 2]
```diff
@@ -620,24 +749,40 @@ cte_tmp_phones as (
...
-select distinct
-    uuid_string() as PhoneId,
-    PhoneNumber,
-    LastUpdateDate
-from CTE_TMP_PHONES
-where not exists (select * from CTE_NOTEXISTS)$$;
+    )
+    
+    select 
+        distinct
```
Comment (ASanchezXtillion) [inline migration_original/ODS1Stage/tables/Base/Phone/spu_translated_Phone.sql:761]: no distinct

**Review by ASanchezXtillion — CHANGES_REQUESTED** [review]
remove the distincts and were gtg

**Review by oajimenez — APPROVED** [review]
LGTM!

**Review by ASanchezXtillion — APPROVED** [review]
LGTM!

**Pull request merged**